### DISCUSSION TITLE
처음시작입니다. 프로젝트를 생성할 수 없다고 나오는 에러는 어떻게 해결할가요?

### DISCUSSION THREAD

**Discussion by bollingerboo** [naverz/zepeto-studio-kor#472] [Issue Report (Legacy)]
![Cap 2022-03-08 19-35-11-290](https://user-images.githubusercontent.com/101181844/157219724-52d77fa5-a270-4c69-90d4-d40fa02d4d9f.png)
 
프로젝트를 생성할 수 없다고 나오네요.  유니티는 당연히 20.3.9f1 이고,  허브로 실행하여 첫단계부터 아무 프로젝트가 생성이 안되네요

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 아래의 유니티 에디터 문제 해결 가이드를 참고하셔서 
라이센스나 설치 폴더 등의 문제인지 확인 부탁드립니다.

https://docs.unity3d.com/kr/2019.4/Manual/TroubleShootingEditor.html
